### ISSUE TITLE
Error when run label_image --graph=/tmp/quantized_graph.pb after quantized model 

### ISSUE THREAD

**Issue by boshanyiqiao** [tensorflow/tensorflow#15089]
tensorflow:1.4.0
I followed the command line from tensorflow/tensorflow/docs_src/performance/quantization.md 
ran the following command is ok and  produce a new model quantized_graph.pb:
  bazel build tensorflow/tools/graph_transforms:transform_graph
  bazel-bin/tensorflow/tools/graph_transforms/transform_graph \
  --in_graph=tensorflow/examples/label_image/data/inception_v3_2016_08_28_frozen.pb \
  --out_graph=/tmp/quantized_graph.pb \
  --inputs=input \
  --outputs=InceptionV3/Predictions/Reshape_1 \
  --transforms='add_default_attributes strip_unused_nodes(type=float, shape="1,299,299,3")
    remove_nodes(op=Identity, op=CheckNumerics) fold_constants(ignore_errors=true)
    fold_batch_norms fold_old_batch_norms quantize_weights quantize_nodes
    strip_unused_nodes sort_by_execution_order'

But there is a problem when  I was running with command:
  bazel build tensorflow/examples/label_image:label_image
  bazel-bin/tensorflow/examples/label_image/label_image \
  --graph=/tmp/quantized_graph.pb    (Else: it worked if I replace quantized_graph.pb with inception_v3_2016_08_28_frozen.pb here.)
The error info is:
  2017-12-04 08:27:02.891427: E tensorflow/core/framework/op_kernel.cc:1142] OpKernel ('op:    "DenseToSparseBatchDataset" device_type: "CPU"') for unknown op: DenseToSparseBatchDataset
2017-12-04 08:27:02.891520: E tensorflow/core/framework/op_kernel.cc:1142] OpKernel ('op: "GroupByWindowDataset" device_type: "CPU"') for unknown op: GroupByWindowDataset
2017-12-04 08:27:02.891562: E tensorflow/core/framework/op_kernel.cc:1142] OpKernel ('op: "IgnoreErrorsDataset" device_type: "CPU"') for unknown op: IgnoreErrorsDataset
2017-12-04 08:27:02.891636: E tensorflow/core/framework/op_kernel.cc:1142] OpKernel ('op: "DatasetToSingleElement" device_type: "CPU"') for unknown op: DatasetToSingleElement
2017-12-04 08:27:02.891686: E tensorflow/core/framework/op_kernel.cc:1142] OpKernel ('op: "SerializeIterator" device_type: "CPU"') for unknown op: SerializeIterator
2017-12-04 08:27:02.891712: E tensorflow/core/framework/op_kernel.cc:1142] OpKernel ('op: "DeserializeIterator" device_type: "CPU"') for unknown op: DeserializeIterator
2017-12-04 08:27:02.891733: E tensorflow/core/framework/op_kernel.cc:1142] OpKernel ('op: "MapAndBatchDataset" device_type: "CPU"') for unknown op: MapAndBatchDataset
2017-12-04 08:27:02.891777: E tensorflow/core/framework/op_kernel.cc:1142] OpKernel ('op: "ParallelInterleaveDataset" device_type: "CPU"') for unknown op: ParallelInterleaveDataset
2017-12-04 08:27:02.891805: E tensorflow/core/framework/op_kernel.cc:1142] OpKernel ('op: "ScanDataset" device_type: "CPU"') for unknown op: ScanDataset
2017-12-04 08:27:02.891825: E tensorflow/core/framework/op_kernel.cc:1142] OpKernel ('op: "SqlDataset" device_type: "CPU"') for unknown op: SqlDataset
2017-12-04 08:27:02.901109: E tensorflow/examples/label_image/main.cc:327] Invalid argument: Node 'InceptionV3/InceptionV3/Conv2d_1a_3x3/BatchNorm/batchnorm/mul_eightbit/input__port__0/reduction_dims': Unknown input node '^input:0'
how should I fix it?

**Comment by arixlin** [comment]
--inputs=input  you need know input param name on graph

**Comment by boshanyiqiao** [comment]
I  am  unfamiliar with tensorflow quantize so that followed the command line and used default params ,what should I add for default --inputs's param ?There are errors when I changed --input='input:1'(sorry, I don't know what should I input param name, do you know default input param name?) , Could you give me a exmple (it can using  the default params) which is working well ?
Thanks for your help!

**Comment by reedwm** [comment]
/CC @petewarden, can you take a look?

**Comment by boshanyiqiao** [comment]
@petewarden I had been modified inception_v3_2016_08_28_frozen.pb  to quantized_graph.pb in tensorflow/examples/label_image/main.cc and label_image.py, then tried to follow command line from https://github.com/petewarden/tensorflow/tree/master/tensorflow/examples/label_image which commands are "bazel build tensorflow/examples/label_image/..." and "bazel-bin/tensorflow/examples/label_image/label_image"，The same problem appears

**Comment by arixlin** [comment]
@boshanyiqiao   tensorboard  --logdir=tensorflow/examples/label_image/data/inception_v3_2016_08_28_frozen.pb   view the graph

**Comment by boshanyiqiao** [comment]
@arixlin 
error info “No dashboards are active for the current data set.” by web page view
$ls tensorflow/examples/label_image/data    (defualt file)
grace_hopper.jpg          inception_v3_2016_08_28_frozen.pb
imagenet_slim_labels.txt  inception_v3_2016_08_28_frozen.pb.tar.gz

Is it and above problem because there is no input file? I use default command and file now, what should I do to fix it?

**Comment by arixlin** [comment]
You need step by step, build  inception_v3_2016_08_28_frozen.pb  before bazel build quantized_graph.pb. Here is model pb link (https://github.com/tensorflow/models/blob/master/research/slim/README.md)   forzen.pb include checkpoint, can't view on tensorboard.

**Comment by arixlin** [comment]
@boshanyiqiao 
https://github.com/tensorflow/models/blob/master/research/slim/README.md
python export_inference_graph.py \
  --alsologtostderr \
  --model_name=inception_v3 \
  --output_file=/tmp/inception_v3_inf_graph.pb

bazel-bin/tensorflow/python/tools/freeze_graph \
  --input_graph=/tmp/inception_v3_inf_graph.pb \
  --input_checkpoint=/tmp/checkpoints/inception_v3.ckpt \
  --input_binary=true --output_graph=/tmp/frozen_inception_v3.pb \
  --output_node_names=InceptionV3/Predictions/Reshape_1

**Comment by zhewang95** [comment]
When I remove the 'quantize_node' option of transform_graph, the transformed graph can be loaded correctly. It seems that there are some bugs in the "quantize_node" option.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.
